daemon/config/config.go

```diff
@@ -514,6 +515,12 @@ func getConflictFreeConfiguration(configFile string, flags *pflag.FlagSet) (*Con
 		return nil, err
 	}
 
+	if runtime.GOOS != "windows" {
+		if _, ok := config.Features["windows-no-dns-proxy"]; ok {
+			return nil, errors.New("feature option 'windows-no-dns-proxy' is only available on Windows")
```

Comment:
Oh! Other comment just made me realise we have a `Config.ValidatePlatformConfig` function with implementations for Linux and Windows, so wondering if this should belong there; https://github.com/moby/moby/blob/484480f56aa38053493e100dbd1c25da0c831df6/daemon/config/config_linux.go#L175-L176

Reply:
Yes, I'd missed that - thank you, I'll move it.